docker-compose.yml

```diff
@@ -26,10 +43,12 @@ services:
       dockerfile: Dockerfile
       target: geoviite-distribution-build
       args:
+        IMAGE_BASE_BACKEND_BUILD: ${IMAGE_BASE_BACKEND_BUILD} # Needed for the geoviite-distribution-build-combiner
+        IMAGE_BASE_DISTRIBUTION: ${IMAGE_BASE_DISTRIBUTION}
         IMAGE_BACKEND: ${IMAGE_BACKEND}
         IMAGE_FRONTEND: ${IMAGE_FRONTEND}
 
-  db-service:
+  postgres-service:
```

Comment:
Kantanimet muutettu jottei mene päällekkäin aiemman devausympäristön kanssa. Samalla standardimpi nimitystyylikin, kun kertoo tarkemmin mikä palvelu on kyseessä (ei minusta kovinkaan epätyypillistä, että olisi useampi erityyppinen kantapalvelu tai kantaa vastaava palvelu).